java/src/main/java/gov/doc/ntia/sigmf/Global.java

```diff
@@ -76,45 +88,94 @@ public class Global implements Serializable {
   @JsonProperty(value = "core:hw", required = false)
   protected String hw;
 
+  // The full filename of the dataset this metadata describes
+  @JsonProperty(value = "core:dataset", required = false)
+  protected String dataset;
+
+  // The number of bytes to ignore at the end of a Non-Conforming dataset file
+  @JsonProperty(value = "core:trailing_bytes", required = false)
+  protected Long trailingBytes;
+
+  // Indicates the metadata is intentionally distributed without the dataset
+  @JsonProperty(value = "core:metadata_only", required = false)
+  protected Boolean metadataOnly;
+
+  // the location of the recording system
+  @Valid
+  @JsonProperty(value = "core:geolocation", required = false)
+  protected GeoJsonPoint geolocation;
+
+  // A list of JSON objects describing extensions used by this recording
+  @Valid
   @JsonProperty(value = "core:extensions", required = false)
-  protected Extensions extensions;
+  protected List<Extension> extensions;
 
+  // The base filename of a SigMF collection with which this recording is associated
+  @JsonProperty(value = "core:collection", required = false)
+  protected String collection;
+
+  @Valid
   @JsonProperty(value = "ntia-sensor:sensor", required = false)
   protected Sensor sensor;
 
+  @Valid
   @JsonProperty(value = "ntia-emitter:emitters", required = false)
   protected List<Emitter> emitters;
 
-  @JsonProperty(value = "ntia-scos:action", required = true)
-  protected Action action;
-
-  @JsonProperty(value = "ntia-scos:schedule", required = true)
-  protected ScheduleEntry schedule;
-
-  @JsonProperty(value = "ntia-location:coordinate_system", required = false)
-  private CoordinateSystem coordinateSystem;
```

Comment:
It looks like coordinateSystem was removed here instead of deprecated. This is ok with me, but checking if this is desired.